migrations/3_pool.js

```diff
@@ -62,9 +147,9 @@ module.exports = async function(deployer) {
         {type: 'uint256', value: '0'}, // zkBobDirectDepositCap
```

Comment:
Limits must be changed before deployment

Reply:
direct deposits being disabled by limits is actually a good idea

Reply:
In fact, the comment was about all limits not only for DD

Reply:
fixed in 692cf51e710ce9a4a73c0700069a413f3b4e1aba